src/content/docs/billing/billing-user-experience/pricing-table-display.mdx

```diff
@@ -0,0 +1,44 @@
+---
+page_id: c47fa1fb-15e1-4dcf-93d9-59df6acdf6da
+title: Pricing table display defaults
+sidebar:
+  order: 4
+relatedArticles:
+  - 37d5ae8e-cfd1-4e5d-8333-387b6967ec23
+  - db047df7-d6b6-4d02-843f-dce6000bacbd
+---
+
+The pricing table that displayed in a particular flow depends on the the [default plan groups and group types](/billing/manage-plans/add-manage-plan-groups/).
+
+## B2C (User-based)
+
+A pricing table is shown when:
+
+- A new user registers
+- An existing user, marked as a billing customer, signs in but isn’t on a plan yet
+
+## B2B (Org-based)
+
+A pricing table is shown when:
+
+- A new organization registers (`is_create_org` parameter)
+- An org member with `org:write:billing` permission signs in and the org is a billing customer but not on a plan yet
+
+## B2B2C (Platforms - uses both)
+
+- Organization plan selection = B2B logic
+- User plan selection = B2C logic
+
+## Override the default plan display
+
+You can override which plan or pricing table is shown using URL parameters. See: [Update code and URLs for billing](/billing/billing-user-experience/add-billing-to-url-sdk/).
+
+## Summary of display ffor plan groups
+
+| Model | Plan group | Pricing table is shown when |
+| --- | --- | --- |
+| B2C | User plan group | New user registers / logs in |
```

Comment:
@clairekinde11 it shouldn't show on log in really unless they have been made a billing customer but not yet put on a plan, I reckon just register is fine